src/components/10-atoms/checkbox/react/story.jsx

```diff
@@ -23,7 +23,7 @@ storiesOf('Components|Atoms/Checkbox/React', module)
       'checkmark',
       'checkmark-inverted',
     ]);
-    const checked = boolean('checked', true);
+
```

Comment:
What about to show that knob, but with your statement that this knob do not make sense here?!

Reply:
The knob is deleted, the diff is red....

Reply:
Yes. That the users are not confused that this knob is missing here (and at the "normal" story it is not) we could tell them that is it deactivated in this case. And why.